src/ui/locales/id.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Alihkan atribusi",
+    "AttributionControl.MapFeedback": "Umpan balik peta",
+    "FullscreenControl.Enter": "Memasuki layar penuh",
+    "FullscreenControl.Exit": "Keluar dari layar penuh",
+    "GeolocateControl.FindMyLocation": "Temukan lokasi saya",
+    "GeolocateControl.LocationNotAvailable": "Lokasi tidak tersedia",
+    "LogoControl.Title": "logo kotak peta",
+    "Map.Title": "Peta",
+    "NavigationControl.ResetBearing": "Setel ulang bantalan ke utara",
```

Comment:
"bearing" in the map is a domain-specific word that doesn't have a literal translation, I think we can keep it as "bearing"
```suggestion
    "NavigationControl.ResetBearing": "Setel ulang bearing ke utara",
```

Reply:
Do you think it's Okay to use `azimut` as a bearing synonym https://github.com/mapbox/mapbox-gl-js/pull/11702#discussion_r1031062282